benchmarks/src/main/scala/zio/GenBenchmarks.scala

```diff
@@ -0,0 +1,82 @@
+package zio
+
+import java.util.concurrent.TimeUnit
+
+import org.openjdk.jmh.annotations._
+import org.scalacheck
+import zio.test.Gen
+import IOBenchmarks._
+
+@State(Scope.Thread)
+@BenchmarkMode(Array(Mode.AverageTime))
+@OutputTimeUnit(TimeUnit.NANOSECONDS)
+class GenBenchmarks {
+
+  @Param(Array("1000"))
+  var size: Int = _
+
+  @Benchmark
+  def zioDouble: Double =
```

Comment:
I think we want to generate a `List[Double]` here.

Reply:
oh, ok! when reading the issue description I thought we want to generate a single `Double` - I'll change that then :)

Reply:
Yes I would add a parameter that defaults to 1000 and generate that many of each type.. WIll help amortize the cost of setup / teardown.

Reply:
Also if you use `List` but construct in the wrong way (e.g. appending), you will end up measuring the performance of `List`. Probably if you rely on each library's facilities to generate a list of something (rather than collecting the results yourself), it will be fine.